app/routes/docs.circulars.corrections/route.mdx

```diff
@@ -28,7 +28,7 @@ Anyone with a [peer endorsement](/docs/circulars/submitting#to-request-a-peer-en
      className="height-auto"
```

Comment:
This needs to be removed or replaced with an updated screenshot.

Reply:
Done:

<img width="1147" height="744" alt="Screenshot 2025-09-09 at 16 29 46" src="https://github.com/user-attachments/assets/a039b5fa-8e76-4c3b-85ec-c861844fde3c" />